src/components/Main/Team/Team.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { useState } from 'react'
+import data from '../../../../data'
+import AddYourself from './AddYourself'
+import PropTypes from 'prop-types';
+
+export const Team = (props) => {
+  const [inputValue, setInputValue] = useState('');
+  const [showComponent, setShowComponent] = useState(false);
+
+  const handleInputChange = (event) => {
+    const newValue = event.target.value;
+    setInputValue(newValue);
+    setShowComponent(true);
+  };
+
+  const {
+    className,
+    team,
+    ...otherProps
```

Comment:
Prawie w każdym komponencie to jest - raczej bym zrezygnował - tak jak pisałęm wcześniej :)